client/src/Reviews.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import AverageRating from './Components/AverageRating';
+import VerifiedReviewsHeader from './Components/VerifiedReviewsHeader';
+import ReviewCard from './Components/ReviewCard';
+
+class Reviews extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: [],
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <div>
+          <h2>Customer Reviews</h2>
+        </div>
```

Comment:
Do all of these need their own divs, or is this for future styling?

Reply:
Good point. 👍
I was putting them in for potential future styling, however they're probably not needed (for the moment at least). On closer inspection of the Groupon page, it doesn't look like they'll require special styling at the App level.